ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/private/with-ep-cos/provision/cucushift-installer-rehearse-ibmcloud-ipi-private-with-ep-cos-provision-chain.yaml

```diff
@@ -36,5 +36,15 @@ chain:
     default: ""
   - name: SERVICE_ENDPOINT_GlobalTagging
     default: ""
+  - name: SERVICE_ENDPOINT_COSConfig
+    default: ""
+  - name: SERVICE_ENDPOINT_GlobalCatalog
+    default: ""
+  - name: SERVICE_ENDPOINT_KeyProtect
+    default: ""
+  - name: SERVICE_ENDPOINT_CIS
```

Comment:
I did not see the default endpoint for CIS defined in eps_default.json, if we are not ready to test it, pls remove it.